.travis.yml

```diff
@@ -32,6 +32,7 @@ install:
   - if [[ "$PYCOSAT" ]]; then
       conda install pycosat=$PYCOSAT;
     fi
+  - pip install ruamel.yaml
```

Comment:
At least conda-smithy does use it (https://github.com/conda-forge/conda-smithy/blob/master/conda_smithy/lint_recipe.py)...